package/packaging.md

```diff
@@ -11,32 +11,7 @@ There is a button on the right hand side menu to flag the package.
 
 
 ### Debian Based Distributions
-The package will need to be repackaged into a deb tarball.
-There are two options: rebuild it from your own debian based machine or build it on the included docker image (recommended).
-
-For either process you will need to update the version number for the package located in the following configuration files:
- * `package/debian/debian/changelog`
-
-
-#### Process for your own machine:
-1. Ensure that you have the following packages installed:
-  * `build-essential`
-  * `devscripts`
-  * `debhelper`
-2. Clone the repo (preferrably into a temporary directory to not mess with any existing branches)
-3. cd into the `package/debian` directory
-4. run the `./debian_build.sh` script
-5. The deb file will be in the `package` directory
-6. upload the deb file to the hosted repo (e.g. the releases page on github)
-
-#### Process for the docker image:
-1. First, fix the version variable in `package/debian/Dockerfile` to the correct value
-  * Don't prepend a 'v', just put the version number
-1. cd into `package/debian`
-2. run `./docker_build.sh`
-3. Enter the name of the folder you want to save the deb file to
-3. Grab some coffee as the image builds and runs
-4. The deb file should be stored in the given folder name gittown_version.deb
-
+The process is now automated from travisCI.
+Now, when a new release is made from github, a new deb file is made and automatically uploaded to the releases page.
```

Comment:
I don't think we need to note that this is `now automated from travisCI`. We will put that in the changelog in the next release. I think this documentation could become:

```
Download the latest deb tarball from the releases page. 
```
I think we should make `releases page` a link as well.

Reply:
This markdown file is made for maintainers on how to build new package files for our users so I think this line is appropriate. A line like `Download the latest tarball` I feel is more of an instruction for end users which belongs better on the website or in the repo readme

Reply:
Then the main thing, I think can be removed is  the use of `now`. We don't need to document the change here and can just be documenting the current state

Reply:
Agreed